plugins/base/src/main/kotlin/signatures/JvmSignatureUtils.kt

```diff
@@ -128,6 +129,12 @@ interface JvmSignatureUtils {
             text(Typography.nbsp.toString())
         }
     }
+
+    fun <T : Documentable> WithExtraProperties<T>.stylesForDeprecated(sourceSetData: SourceSetData): Set<TextStyle> =
```

Comment:
Function name is a little bit misleading IMO 🤔 
It says "stylesForDeprecated", but the inspects the source extras to see whether or not it is even deprecated.

Suggestion: `stylesIfDeprecated`

Reply:
Ok, will change